scripts/create-all-tests.js

```diff
@@ -38,7 +38,7 @@ async function main() {
   const VALIDATE_CHECK = !!args.validate;
 
   const scriptsDirectory = path.dirname(__filename);
-  const rootDirectory = scriptsDirectory.split('scripts')[0];
+  const rootDirectory = path.join(scriptsDirectory, '../');
```

Comment:
Don't think you need the trailing `/`, just the `..`. probably better for cross environment too

Reply:
Thanks for that note!

Reply:
Done